src/lib/DataConnectStore.js

```diff
@@ -59,6 +59,22 @@ export default class DataConnectStore {
     return useCase.connectors.map(c1 => this.find(c2 => c1.slug === c2.slug))
   }
 
+  // Fetch all accounts and updates their matching connectors
+  fetchAllAccounts (index) {
+    if (!index && this.accountsIndex) index = this.accountsIndex
```

Comment:
@gregorylegarec I agree but in this case, we will always query the index ID to list accounts instead of store it to make less useless server requests.